calcom/cal.com
# fix: broken google account avatar fix

## What does this PR do?
seems like previoulsy avatar url was broken when signing up and loging in with google... turns out CAL_URL was added to the avatar url that was coming from google account. so I made small changes to check if avatar url starts with slash, only then prepend CAL_URL to it.

before:
![image](https://github.com/calcom/cal.com/assets/76877078/310a589d-2d9f-416d-a02d-62190eb621be)

after:
![image](https://github.com/calcom/cal.com/assets/76877078/0b5101e4-114a-4dcb-92f6-6593afe23104)


## Type of change

- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?

To confirm the issue, sign up with google then login and go to the profile page (https://app.cal.com/settings/my-account/profile)
and profile picture on the top of sidebar as well as profile picture on the upload avatar modal was broken, as shown in screenshot 1


## Mandatory Tasks
